MudBlazor solution won't build in GitHub CodeSpaces or on new Apple silicon Macs

### Bug type

Other

### Component name

_No response_

### What happened?

Currently due to WebCompiler our solution won't build in GitHub CodeSpaces.
CodeSpaces is a fantastic technology we should support.  Its basically almost full vscode in the browser.

In addition WebCompiler also blocks building on ARM processors such as the new Apple Silicon MacBooks.

I would like to fix this by updating the build system that generates the static assets to use MSBuild Tasks.

I plan to use
https://github.com/Vampire2008/Citizen17.DartSass
as a basis for the sass (underlying build is done by DartSass) and 
https://github.com/Taritsyn/JSMin.NET
for the JS minification.



### Expected behavior

MudBlazor should build without error on new Macs and GitHub CodeSpaces.

### Reproduction link

None

### Reproduction steps

1. dotnet build


### Relevant log output

```shell
/home/codespace/MudBlazor/src/MudBlazor/MudBlazor.csproj(80,5): error MSB3073: The command "dotnet webcompiler ./Styles/MudBlazor.scss -c excubowebcompiler.json" exited with code 150. [TargetFramework=net7.0]
```

### Version (bug)

Any

### Version (working)

Never

### What browsers are you seeing the problem on?

Other

### On what operating system are you experiencing the issue?

Other

### Pull Request

- [x] I would like to do a Pull Request

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

--- a/src/MudBlazor.Docs/.config/dotnet-tools.json
+++ b/src/MudBlazor.Docs/.config/dotnet-tools.json
@@ -3,7 +3,7 @@
   "isRoot": true,
   "tools": {
     "excubo.webcompiler": {
-      "version": "2.7.12",
+      "version": "3.5.44",
       "commands": [
         "webcompiler"
       ]

--- a/src/MudBlazor.Docs/excubowebcompiler.json
+++ b/src/MudBlazor.Docs/excubowebcompiler.json
@@ -42,8 +42,7 @@
     "Sass": {
       "IndentType": "Space",
       "IndentWidth": 2,
-      "OutputStyle": "Nested",
-      "Precision": 5,
+      "OutputStyle": "Expanded",
       "RelativeUrls": true,
       "LineFeed": "Lf",
       "SourceMap": false

--- a/src/MudBlazor/.config/dotnet-tools.json
+++ b/src/MudBlazor/.config/dotnet-tools.json
@@ -3,7 +3,7 @@
   "isRoot": true,
   "tools": {
     "excubo.webcompiler": {
-      "version": "2.7.12",
+      "version": "3.5.44",
       "commands": [
         "webcompiler"
       ]

--- a/src/MudBlazor/excubowebcompiler.json
+++ b/src/MudBlazor/excubowebcompiler.json
@@ -42,8 +42,7 @@
     "Sass": {
       "IndentType": "Space",
       "IndentWidth": 2,
-      "OutputStyle": "Nested",
-      "Precision": 5,
+      "OutputStyle": "Expanded",
       "RelativeUrls": true,
       "LineFeed": "Lf",
       "SourceMap": false

